docs/docs/getting-started.md

```diff
@@ -27,6 +27,12 @@ ReactDOM.render(
 );
 ```
 
+If you are using webpack to bundle your dependencies: 
+
+```sh
+$ npm install --save react react-dom babelify babel-preset-react
```

Comment:
We have this exact line just below here (though it's in the browserify block). Is that what you intended?

As a side note, we could definitely make this section a little clearer, adding a shared dependency section (eg installing react-dom) and then a section with specific instructions for browserify and another for browserify.